web-components/src/components/combobox/ComboBox.ts

```diff
@@ -149,6 +150,16 @@ export namespace ComboBox {
     private multiSelected: number[] = [];
     private customContent: Element[] = [];
 
+    private notifylistitemscounttoparent() {
```

Comment:
rename notifylistitemscounttoparent to notifySearchResultCount

Reply:
Done